packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts

```diff
@@ -172,6 +172,9 @@ export class ForceLightningLwcStartExecutor extends SfdxCommandletExecutor<{}> {
     if (!serverStarted && !cancellationToken.isCancellationRequested) {
       let message = nls.localize('force_lightning_lwc_start_failed');
 
+      if (exitCode === 112) {
+        message = nls.localize('force_lightning_lwc_inactive_scratch_org');
```

Comment:
I noticed in the PR that shelby opened up she added some more error handling logic for the specific exit code in `execution.stderrSubject.subscribe` is that something we have to do here? Or is that exit code automatically returned when the scratch org is expired?

Reply:
Good question.

My changes were a bit different - I was looking for specific text printed to stderr and then called this same logic with the new error code. In my case, the process was not exiting with the error code I wanted to see - which is why I was examining stderr. If in the future we received the correct error code via processExitSubject, we would still process it the same way in this handleErrors method. 

It sounds like this code execution is hitting processExitSubject with the exit code 112.

Reply:
Ah okay that makes sense, thanks :)

Reply:
Thanks @sfsholden . @AnanyaJha Yeah I was going to let lwc-dev-server return 112 when the scratch org expires. FYI the correspondent PR in lwc-dev-server: https://github.com/forcedotcom/lwc-dev-server/pull/189